components/library/libgksu/Makefile

```diff
@@ -10,13 +10,16 @@
 
 #
```

Comment:
If libgksu needs to be rebuilt after the libgtop update, you'll need to put it into a separate PR otherwise libgksu will be build against the old libgtop (new packages get installed after creating all packages of a PR).

Reply:
There is quite a bit that needs rebuilding. Honestly this should be something for the build system to handle correctly.
For now I just added the non mate ones as those will be rebuild because of the upgrade to 1.24 anyway.